### PR TITLE
python@3.13 python-gdbm@3.13 python-tk@3.13

### PR DESCRIPTION
**Pull request by lukeshingles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
python3.13 released on October 7th.

CI failures probably unrelated to this PR:
- [x] codecov-cli: fixed by #191763
- [x] dependency-check fixed by #191474
- [x] embulk test fixed by #191470
- [x] gorilla-cli fixed by #192048 

CI failures probably caused by demoting python@3.12 to altinstall
- [x] afl++
- [x] bandcamp-dl
- [x] buku 
- [x] chapel
- [ ] llvm, llvm@16. lldb python has wrong sys.path on Linux, need alternative to #192626 fix. Can be fixed upgrading formulae to python3.13, but CI time will be very long.
- [x] nanobind
- [x] restview
- [x] streamlink
- [x] swig (fixed by #192808)
- [x] woof

### CODE REVIEW

**Comment by lukeshingles** [comment]
I'm hitting 1hr timeouts on the dep runners (could a maintainer please add a label to extend this?). I don't expect the changes to the python@3.12 formula to break any dependents, since it's just being demoted to an altinstall and no longer being symlinked to pip, python, etc commands.

**Comment by lukeshingles** [comment]
@branchvincent @cho-m @p-linnane I think you're often working with the python formulae. Could I get a long-build label added (and any code comments) to get ready for python@3.13 final on October 1st?

**Comment by branchvincent** [comment]
Generally looks good at a glance, nice work! Let's wait for 3.12.6 to be merged first in #183866 and then we can run this with a long timeout

**Comment by lukeshingles** [comment]
> Generally looks good at a glance, nice work! Let's wait for 3.12.6 to be merged first in #183866 and then we can run this with a long timeout

Ok, 3.12.6 is in and the branch is up to date now.

**Comment by lukeshingles** [comment]
I think a long-build is needed.

**Comment by carlocab** [comment]
I'd split out the `gorilla-cli`, `embulk`, and `dependency-check` changes to other PRs since they don't require the changes here to make sense.

**Comment by lukeshingles** [comment]
Getting some weird CI errors for dependent forumulae that are not changed by the PR. Example:
```sh
  ==> Downloading https://ghcr.io/v2/homebrew/core/flawfinder/manifests/2.0.19_1-2
  Already downloaded: /home/linuxbrew/.cache/Homebrew/downloads/46aa6b91a7ad88c12b2c7d857dbf6f9d10e719e33111085123e08403a26c0457--flawfinder-2.0.19_1-2.bottle_manifest.json
  ==> Fetching flawfinder
  ==> Downloading https://ghcr.io/v2/homebrew/core/flawfinder/blobs/sha256:29f9397a8cee6ea519559666eca9fd4323259bc449543e8bc0f4afd30aeaceaa
  Already downloaded: /home/linuxbrew/.cache/Homebrew/downloads/59c2c74892e154be4dd9ca8971f97b058896152bd270be81044ab8e9d191b5ff--flawfinder--2.0.19_1.x86_64_linux.bottle.2.tar.gz
  ==> Verifying attestation for flawfinder
  Error: The bottle for flawfinder has an invalid build provenance attestation.
  This may indicate that the bottle was not produced by the expected
  tap, or was maliciously inserted into the expected tap's bottle
  storage.
  Additional context:
  no attestation matches subject: 59c2c74892e154be4dd9ca8971f97b058896152bd270be81044ab8e9d191b5ff--flawfinder--2.0.19_1.x86_64_linux.bottle.2.tar.gz
  Error: The bottle for flawfinder has an invalid build provenance attestation.
  
  This may indicate that the bottle was not produced by the expected
  tap, or was maliciously inserted into the expected tap's bottle
  storage.
  
  Additional context:
  
  no attestation matches subject: 59c2c74892e154be4dd9ca8971f97b058896152bd270be81044ab8e9d191b5ff--flawfinder--2.0.19_1.x86_64_linux.bottle.2.tar.gz
```

File: Formula/p/python@3.13.rb
Comment (carlocab) [inline Formula/p/python@3.13.rb:10]:
```suggestion
    regex(%r{href=.*?v?(3\.13(?:\.\d+)*)/?["' >]}i)
```

File: Formula/p/python@3.13.rb
Comment (carlocab) [inline Formula/p/python@3.13.rb:232]: Don't think we want to keep this line.

File: Formula/p/python@3.13.rb
Comment (carlocab) [inline Formula/p/python@3.13.rb:84]: Yes, move this to formula-patches please.

File: Formula/s/swig.rb
Comment (carlocab) [inline Formula/s/swig.rb:1]: You could probably split this out into a separate PR.

File: Formula/p/python@3.13.rb
Comment (lukeshingles) [reply Formula/p/python@3.13.rb:84]: Made PR #192808

File: Formula/s/swig.rb
Comment (lukeshingles) [reply Formula/s/swig.rb:1]: Made PR #192808

File: Formula/l/lxi-tools.rb
Comment (carlocab) [inline Formula/l/lxi-tools.rb:1]: I think this has been fixed on `master`; you might be able to remove it.

**Comment by henryiii** [comment]
Final release is out!

**Comment by johng** [comment]
Will there be a python3.13t version as part of this change?

**Comment by branchvincent** [comment]
Build timed out, let's drop `llvm` and just follow up separately after this is merged

**Comment by branchvincent** [comment]
Thanks a ton for your work here @lukeshingles!! We couldn't have shipped this as fast without you, thank you 🍻 

---

Follow ups:
- [ ] `llvm`
- [ ] `llvm@16`
- [x] `internetarchive`: network flake?
- [x] `waybackpy`: network flake?

File: Formula/p/python@3.13.rb
Comment (branchvincent) [inline Formula/p/python@3.13.rb:172]: I missed this earlier, but we can fix this after

**Review by branchv — APPROVED** [review]

File: Formula/p/python@3.13.rb
Comment (lukeshingles) [reply Formula/p/python@3.13.rb:181]: Sorry, dumb mistake on my part there when I switched between embedded and external patch.

**Pull request merged**
